packets1/property_message_id_test.go

```diff
@@ -0,0 +1,19 @@
+package packets1
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMessageID(t *testing.T) {
+	msgID := uint16(1234)
+
+	pkt1 := NewPubrec()
+	pkt1.SetMessageID(msgID)
+	pkt2 := NewPubrel()
+	pkt2.CopyMessageID(pkt1)
+
+	assert.Equal(t, msgID, pkt2.messageID)
```

Comment:
It is always a red flag when a test accesses a private field, it couples it tightly with the implementation. Given that `MessageID()` is a simple getter, I’d just skip this assertion.

Reply:
You are right, I will rather use `MessageID()`.